tico/quantization/algorithm/gptq/gptq.py

```diff
@@ -181,6 +181,23 @@ def fasterquant(
         if actorder:
             Q = Q[:, invperm]
 
+        if isinstance(self.layer, nn.Conv2d):
+            if groupsize == -1:  # TODO support groupsize != -1
+                Q[:, dead] = quantize(
+                    self.layer.weight.flatten(1)[:, dead],
+                    self.quantizer.scale,
+                    self.quantizer.zero,
+                    self.quantizer.maxq,
+                )
+        else:
+            if groupsize == -1:  # TODO support groupsize != -1
+                Q[:, dead] = quantize(
+                    self.layer.weight[:, dead],
+                    self.quantizer.scale,
+                    self.quantizer.zero,
+                    self.quantizer.maxq,
+                )
+
```

Comment:
Could you add `assert groupsize == -1` to prevent undefined behavior of groupsize != -1 case?

Reply:
> Could you add `assert groupsize == -1` to prevent undefined behavior of groupsize != -1 case?

ok.